src/Contracts/WebAuthnAuthenticatable.php

```diff
@@ -34,14 +34,14 @@ public function disableAllCredentials(string ...$except): void;
      * Makes an instance of a WebAuthn Credential attached to this user.
      *
      * @param  array  $properties
-     * @return \Laragear\WebAuthn\Models\WebAuthnCredential
+     * @return WebAuthnCredential
      */
     public function makeWebAuthnCredential(array $properties): WebAuthnCredential;
 
     /**
      * Returns a queryable relationship for its WebAuthn Credentials.
      *
-     * @return \Illuminate\Database\Eloquent\Relations\MorphMany&\Laragear\WebAuthn\Models\WebAuthnCredential
+     * @return MorphMany
```

Comment:
Why this? While

```php
* @return \Illuminate\Database\Eloquent\Relations\MorphMany&\Laragear\WebAuthn\Models\WebAuthnCredential
```

has surely been incorrect

```php
@return MorphMany
```

is, too. I assume

```suggestion
     * @return MorphMany<\Laragear\WebAuthn\Models\WebAuthnCredential>
```

is what was actually meant? The relation classes need the target of the relation as template parameter. I haven't checked the code thoroughly nor did I run tests, but this is what I suspect from code review.

Reply:
This helped.